shell/src/main/java/alluxio/cli/LogLevel.java

```diff
@@ -0,0 +1,257 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli;
+
+import alluxio.Constants;
+import alluxio.client.block.AlluxioBlockStore;
+import alluxio.client.block.BlockWorkerInfo;
+import alluxio.util.network.HttpUtils;
+import alluxio.util.network.NetworkAddressUtils;
+import alluxio.util.network.NetworkAddressUtils.ServiceType;
+import alluxio.wire.LogInfo;
+import alluxio.wire.WorkerNetAddress;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.apache.commons.lang.StringUtils;
+import org.apache.http.client.utils.URIBuilder;
+
+import javax.annotation.concurrent.NotThreadSafe;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Set and get the specify target specify log's level.
+ */
+@NotThreadSafe
+public final class LogLevel {
+  private static final String LOG_LEVEL = "logLevel";
+  private static final String TARGET_SEPARATOR = ",";
+  private static final String TARGET_OPTION_NAME = "target";
+  private static final Option TATGET_OPTION =
+      Option.builder()
+          .required(false)
+          .longOpt(TARGET_OPTION_NAME)
+          .hasArg(true)
+          .desc("<master|workers|host:webPort>."
```

Comment:
could we make the third option `host:webport:role`? Then users could set log levels for multiple masters in HA setups